.env.apicast

```diff
@@ -1,3 +0,0 @@
-THREESCALE_PORTAL_ENDPOINT=http://master-account-admin.example.com:3000
```

Comment:
Good catch on this file, it was used in a really old Dockerfile but we forgot to remove this
For the history:

- The former Dockerfile and Makefile had the roles to build a local integration environment through docker-compose
- We removed it because it was not maintained and was hard to maintain

My opinion on local integration. I prefer a shared project that will spin up the whole 3scale or part of it in another project. And each service could point either to the container image or a local git version.

Something like in another project:

```
# Edit some configuration file to point to your local git directories or container
# Then
make 3scale
```

And it will spin all the components respectively for integration
A bit like https://github.com/guicassolato/porta-dev-tools

Reply:
> I am not against keeping the .env.example but clearly simplify it.

Removed because you said that you want to remove `.env` support some day. So I thought not to promote its usage.

Reply:
I mean dotenv and dotenv-rails ... ;)
FYI, you can still use `.env` with autoenv which is a shell integration

Reply:
IMO whoever wants to use `autoenv` or something else, will know how to do. I tried to document env variables so that anybody can set them as they like. Makes sense? Or do you want the file back?